### PR TITLE
[TRANSLATION] Improve Korean (ko-KR) translation

### PR DESCRIPTION
**Pull request by njlyf2011**
Improve Korean translation  (Explorer Shell and SHELL32).

### CODE REVIEW

File: base/shell/explorer/lang/ko-KR.rc [hunk 1]
```diff
@@ -84,11 +85,11 @@ END
 
 IDD_TASKBARPROP_STARTMENU DIALOGEX 0, 0, 252, 218
 STYLE DS_SHELLFONT | DS_MODALFRAME | WS_POPUP | WS_VISIBLE | WS_CAPTION | WS_SYSMENU
-CAPTION "Start Menu"
+CAPTION "시작 메뉴"
 FONT 9, "굴림", 0, 0, 0x1
 BEGIN
     CONTROL "", IDC_TASKBARPROP_STARTMENU_BITMAP, "Static", SS_BITMAP | SS_SUNKEN,7, 7, 183, 108
-    AUTORADIOBUTTON "&Start menu", IDC_TASKBARPROP_STARTMENU, 7, 135, 105, 10
+    AUTORADIOBUTsTON "시작 메뉴(&S)", IDC_TASKBARPROP_STARTMENU, 7, 135, 105, 10
```
Comment (binarymaster) [inline base/shell/explorer/lang/ko-KR.rc:92]: Misclick here :)
```suggestion
    AUTORADIOBUTTON "시작 메뉴(&S)", IDC_TASKBARPROP_STARTMENU, 7, 135, 105, 10
```

File: base/shell/explorer/lang/ko-KR.rc [hunk 2]
```diff
@@ -1,4 +1,5 @@
 /* Korean translation by manatails007 - Seungju Kim */
+/* Improved by Luke Luo <njlyf2011@hotmail.com> */
```
Comment (binarymaster) [inline base/shell/explorer/lang/ko-KR.rc:2]: Let's use standard header here:
```c
/*
 * PROJECT:     ReactOS Explorer
 * LICENSE:     LGPL-2.1-or-later (https://spdx.org/licenses/LGPL-2.1-or-later)
 * PURPOSE:     Korean resource file
 * TRANSLATORS: Copyright 2008 Seungju Kim aka manatails007 <admin@manateeshome.com>
 *              Copyright 2022 Luke Luo <njlyf2011@hotmail.com>
 */
```

**Comment by njlyf2011** [comment]
Thanks.
I will improve them later.

**Comment by njlyf2011** [comment]
@katahiromz Excuse me, could you review my translation?

**Comment by njlyf2011** [comment]
@katahiromz Excuse me, could you review my translation?

**Review by katahiromz — COMMENTED** [review]
I am not Korean so I can't understand Korean language. You might have to translate all text.

**Comment by katahiromz** [comment]
```txt
2022-07-12T05:01:45.9330326Z    Creating library dll\win32\vbscript\vbscript.lib and object dll\win32\vbscript\vbscript.exp
2022-07-12T05:01:45.9334746Z global.c.obj : error LNK2019: unresolved external symbol round referenced in function Global_Round
2022-07-12T05:01:45.9335541Z dll\win32\vbscript\vbscript.dll : fatal error LNK1120: 1 unresolved externals
```
This compilation error is not relevant to this PR.

**Comment by njlyf2011** [comment]
I'm a Korean learner, so I can't translate the entire text.  But I can be sure that the text that has been translated is correct.

**Comment by njlyf2011** [comment]
> I am not Korean so I can't understand Korean language. You might have to translate all text.

I'm a Korean learner, so I can't translate the entire text.  But I can be sure that the text that has been translated is correct.

You are an active developer from Eastern Asian, and you have fixed the Korean font problem of ReactOS, so I ask you.

File: dll/win32/shell32/lang/ko-KR.rc [hunk 3]
```diff
@@ -153,129 +160,129 @@ IDD_ABOUT_AUTHORS DIALOGEX 35, 90, 235, 85
...
+    DEFPUSHBUTTON "확인", IDOK, 36, 70, 59, 14, WS_TABSTOP
+    PUSHBUTTON "취소", IDCANCEL, 98, 70, 59, 14, WS_TABSTOP
+    PUSHBUTTON "찾아보기(&B)...", 12288, 160, 70, 59, 14, WS_TABSTOP
 END
 
 IDD_SHORTCUT_PROPERTIES DIALOGEX 0, 0, 235, 215
 STYLE DS_SHELLFONT | WS_CHILD | WS_CAPTION
-CAPTION "Shortcut"
+CAPTION "바로 가기"
 FONT 9, "굴림", 0, 0, 0x0
```
Comment (katahiromz) [inline dll/win32/shell32/lang/ko-KR.rc:184]:
```suggestion
FONT 9, "굴림", 0, 0, 0x1
```
The 5th parameter of `FONT` is a charset.

File: dll/win32/shell32/lang/ko-KR.rc [hunk 4]
```diff
@@ -153,129 +160,129 @@ IDD_ABOUT_AUTHORS DIALOGEX 35, 90, 235, 85
...
+    PUSHBUTTON "대상 찾기(&F)...", IDC_SHORTCUT_FIND, 9, 172, 70, 14, ES_LEFT
+    PUSHBUTTON "아이콘 변경(&C)...", IDC_SHORTCUT_CHANGE_ICON, 84, 172, 70, 14, ES_LEFT
+    PUSHBUTTON "고급(&D)...", IDC_SHORTCUT_ADVANCED, 159, 172, 70, 14, ES_LEFT
 END
 
 IDD_SHORTCUT_EXTENDED_PROPERTIES DIALOGEX 0, 0, 230, 150
 STYLE DS_SHELLFONT | DS_MODALFRAME | WS_POPUP | WS_CAPTION
-CAPTION "Extended Properties"
+CAPTION "고급 속성"
 FONT 9, "굴림", 0, 0, 0x0
```
Comment (katahiromz) [inline dll/win32/shell32/lang/ko-KR.rc:212]:
```suggestion
FONT 9, "굴림", 0, 0, 0x1
```

File: dll/win32/shell32/lang/ko-KR.rc [hunk 5]
```diff
@@ -153,129 +160,129 @@ IDD_ABOUT_AUTHORS DIALOGEX 35, 90, 235, 85
...
-    PUSHBUTTON "Abort", IDCANCEL, 120, 124, 50, 15, WS_VISIBLE
+    PUSHBUTTON "확인", IDOK, 63, 124, 50, 15, WS_VISIBLE
+    PUSHBUTTON "중단하다", IDCANCEL, 120, 124, 50, 15, WS_VISIBLE
 END
 
 IDD_FOLDER_PROPERTIES DIALOGEX 0, 0, 240, 205
 STYLE DS_SHELLFONT | WS_CHILD | WS_CAPTION
-CAPTION "General"
+CAPTION "일반"
 FONT 9, "굴림", 0, 0, 0x0
```
Comment (katahiromz) [inline dll/win32/shell32/lang/ko-KR.rc:226]:
```suggestion
FONT 9, "굴림", 0, 0, 0x1
```

File: dll/win32/shell32/lang/ko-KR.rc [hunk 6]
```diff
@@ -298,7 +305,7 @@ END
 
 IDD_DRIVE_PROPERTIES DIALOGEX 0, 0, 240, 230
 STYLE DS_SHELLFONT | WS_CHILD | WS_DISABLED | WS_CAPTION
-CAPTION "General"
+CAPTION "일반"
 FONT 9, "굴림", 0, 0, 0x0
```
Comment (katahiromz) [inline dll/win32/shell32/lang/ko-KR.rc:309]:
```suggestion
FONT 9, "굴림", 0, 0, 0x1
```

File: dll/win32/shell32/lang/ko-KR.rc [hunk 7]
```diff
@@ -331,7 +338,7 @@ END
 
 IDD_DRIVE_TOOLS DIALOGEX 0, 0, 240, 230
 STYLE DS_SHELLFONT | WS_CHILD | WS_DISABLED | WS_CAPTION
-CAPTION "Tools"
+CAPTION "도구"
 FONT 9, "굴림", 0, 0, 0x0
```
Comment (katahiromz) [inline dll/win32/shell32/lang/ko-KR.rc:342]:
```suggestion
FONT 9, "굴림", 0, 0, 0x1
```

File: dll/win32/shell32/lang/ko-KR.rc [hunk 8]
```diff
@@ -350,14 +357,14 @@ END
...
+CAPTION "하드웨어"
 FONT 9, "굴림", 0, 0, 0x0
 BEGIN
 END
 
 IDD_RUN_AS DIALOGEX 0, 0, 240, 190
 STYLE DS_SHELLFONT | WS_CHILD | WS_DISABLED | WS_CAPTION
-CAPTION "Run As"
+CAPTION "다음 계정으로 실행"
 FONT 9, "굴림", 0, 0, 0x0
```
Comment (katahiromz) [inline dll/win32/shell32/lang/ko-KR.rc:368]:
```suggestion
FONT 9, "굴림", 0, 0, 0x1
```

File: dll/win32/shell32/lang/ko-KR.rc [hunk 9]
```diff
@@ -407,7 +414,7 @@ END
 
 IDD_FOLDER_OPTIONS_GENERAL DIALOGEX 0, 0, 264, 256
 STYLE DS_SHELLFONT | WS_POPUP | WS_CAPTION
-CAPTION "General"
+CAPTION "일반"
 FONT 9, "굴림", 0, 0, 0x0
```
Comment (katahiromz) [inline dll/win32/shell32/lang/ko-KR.rc:418]:
```suggestion
FONT 9, "굴림", 0, 0, 0x1
```

**Comment by binarymaster** [comment]
I have performed a grep for Korean translators in ReactOS, so far we have only two:
- Seungju Kim - @manatails on GitHub
- YunSong Hwang - perhaps a Wine translator, does not seem to have GitHub account

@manatails if you are available, please review this translation PR.

File: dll/win32/shell32/lang/ko-KR.rc [hunk 10]
```diff
@@ -625,16 +632,16 @@ END
...
+CAPTION "아이콘 변경"
 FONT 9, "굴림", 400, 0, 0x1
 BEGIN
-    LTEXT "Look for icons in this file:", -1, 7, 7, 179, 10
-    PUSHBUTTON "Browse...", IDC_BUTTON_PATH, 135, 17, 50, 14
+    LTEXT "파일에서 이 아이콘 찾기(&L):", -1, 7, 7, 179, 10
+    PUSHBUTTON "찾아보기(&B)...", IDC_BUTTON_PATH, 135, 17, 50, 14
     EDITTEXT IDC_EDIT_PATH, 7, 17, 123, 14, ES_AUTOHSCROLL
-    LTEXT "Select an icon from the list below:", -1, 7, 36, 179, 10
+    LTEXT "아래 목록에서 아이콘 선:", -1, 7, 36, 179, 10
```
Comment (manatails) [inline dll/win32/shell32/lang/ko-KR.rc:641]:
```suggestion
    LTEXT "아래 목록에서 아이콘 선택:", -1, 7, 36, 179, 10
```

File: dll/win32/shell32/lang/ko-KR.rc [hunk 11]
```diff
@@ -153,129 +160,129 @@ IDD_ABOUT_AUTHORS DIALOGEX 35, 90, 235, 85
...
 BEGIN
     ICON IDI_SHELL_EXTENDED_PROPERTIES, IDI_SHELL_EXTENDED_PROPERTIES, 5, 5, 21, 20, SS_ICON
     LTEXT "Choose the advanced properties you want for this shortcut.", -1, 5, 30, 210, 10
     CHECKBOX "Run with different credentials", IDC_SHORTEX_RUN_DIFFERENT, 25, 50, 150, 10
     LTEXT "This option can allow you to run the this shortcut as another user, or continue as yourself while protecting your computer and data from unauthorized program activity.", -1, 50, 60, 175, 40
     CHECKBOX "Run in seperate memory space", IDC_SHORTEX_RUN_SEPARATE, 25, 100, 150, 10, WS_DISABLED
-    PUSHBUTTON "OK", IDOK, 63, 124, 50, 15, WS_VISIBLE
-    PUSHBUTTON "Abort", IDCANCEL, 120, 124, 50, 15, WS_VISIBLE
+    PUSHBUTTON "확인", IDOK, 63, 124, 50, 15, WS_VISIBLE
+    PUSHBUTTON "중단하다", IDCANCEL, 120, 124, 50, 15, WS_VISIBLE
```
Comment (manatails) [inline dll/win32/shell32/lang/ko-KR.rc:220]:
```suggestion
    PUSHBUTTON "중단", IDCANCEL, 120, 124, 50, 15, WS_VISIBLE
```

File: dll/win32/shell32/lang/ko-KR.rc [hunk 12]
```diff
@@ -153,129 +160,129 @@ IDD_ABOUT_AUTHORS DIALOGEX 35, 90, 235, 85
...
+    LTEXT "시작 위치(&S):", IDC_SHORTCUT_START_IN, 8, 98, 68, 10
     EDITTEXT IDC_SHORTCUT_START_IN_EDIT, 79, 96, 150, 14, ES_AUTOHSCROLL
-    LTEXT "Shortcut &key:", IDC_SHORTCUT_KEY, 8, 117, 68, 10
+    LTEXT "바로 가기 키(&K):", IDC_SHORTCUT_KEY, 8, 117, 68, 10
     CONTROL "", IDC_SHORTCUT_KEY_HOTKEY, "msctls_hotkey32", WS_VISIBLE | WS_CHILD | WS_BORDER | WS_TABSTOP, 79, 115, 150, 14
-    LTEXT "&Run:", IDC_SHORTCUT_RUN, 8, 136, 68, 10
+    LTEXT "실행(&R):", IDC_SHORTCUT_RUN, 8, 136, 68, 10
     COMBOBOX IDC_SHORTCUT_RUN_COMBO, 79, 134, 150, 14, CBS_DROPDOWNLIST | WS_CHILD | WS_VISIBLE | WS_VSCROLL | WS_TABSTOP
-    LTEXT "C&omment:", IDC_SHORTCUT_COMMENT, 8, 154, 68, 10
+    LTEXT "아이콘 변경(&C)...", IDC_SHORTCUT_COMMENT, 8, 154, 68, 10
```
Comment (manatails) [inline dll/win32/shell32/lang/ko-KR.rc:202]:
```suggestion
    LTEXT "설명(&C)...", IDC_SHORTCUT_COMMENT, 8, 154, 68, 10
```

**Review by manatails — CHANGES_REQUESTED** [review]
I fixed some minor problems. It will be good for merge.

**Comment by njlyf2011** [comment]
ok.

File: dll/win32/shell32/lang/ko-KR.rc [hunk 13]
```diff
@@ -1,3 +1,10 @@
+/*
+ * PROJECT:     ReactOS Shell32
+ * LICENSE:     LGPL-2.1+ (https://spdx.org/licenses/LGPL-2.1+)
+ * PURPOSE:     Korean resource file
+ * TRANSLATORS: Luke Luo <njlyf2011@hotmail.com>
+ */
```
Comment (binarymaster) [inline dll/win32/shell32/lang/ko-KR.rc:6]:
```suggestion
/*
 * PROJECT:     ReactOS Shell32
 * LICENSE:     LGPL-2.1-or-later (https://spdx.org/licenses/LGPL-2.1-or-later)
 * PURPOSE:     Korean resource file
 * TRANSLATORS: Copyright 2022 Luke Luo <njlyf2011@hotmail.com>
 */
```

**Pull request merged**